homeassistant/components/litejet/trigger.py

```diff
@@ -42,15 +44,15 @@ async def async_attach_trigger(
 ) -> CALLBACK_TYPE:
     """Listen for events based on configuration."""
     trigger_data = trigger_info["trigger_data"]
-    number = config.get(CONF_NUMBER)
+    number = cast(int, config.get(CONF_NUMBER))
```

Comment:
This is wrong. It seems CONF_NUMBER is required:
```suggestion
    number = cast(int, config[CONF_NUMBER])
```